Week3/week3-solution.txt

```diff
@@ -0,0 +1,178 @@
+Write a python script to perform CRUD operation on following table of
+�ESM.db� database.
+Employee ( eid, ename, dept, basic, branch). Eid must be primary key.
+Department : Account, Inventory, IT, HR.
+Peform Following Operation.
+
+1. Create table
+->
+Import sqlite3
+Conn = connect('C:\\sqlite\\ESM.db')
+Cur = conn.cursor()
+Q = "create table Employee
+(
+Eid integer primary key,
+Ename text,
+Dept text,
+Basic integer,
+Branch text
+);"
+Cur.execute(Q)
+Conn.commit()
+Conn.close()
+
+2. Insert 5 Records directly, 5 records using tuple and 5 records using
+taking input from user
+->
+Import sqlite3

```

Comment:
Do we need to import sqlite3 , connect with database and create a cursor object everytime or only at once..?